.env.example

```
@@ -9,4 +9,8 @@ NEXT_PUBLIC_SUPABASE_PUBLISHABLE_KEY=your-supabase-publishable-key
 SUPABASE_SERVICE_ROLE_KEY=your-supabase-service-role-key
 
 # CSVインポートで作成されるユーザーのデフォルトパスワード
-DEFAULT_USER_PASSWORD=P@ssw0rd
\ No newline at end of file
+DEFAULT_USER_PASSWORD=P@ssw0rd
+
+# Guest Login
+GUEST_USER_EMAIL=guest@example.com
+GUEST_USER_PASSWORD=P@ssw0rd
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)